adapters/mobfoxpb/mobfoxpb.go

```diff
@@ -0,0 +1,137 @@
+package mobfoxpb
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/buger/jsonparser"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type MobfoxpbAdapter struct {
+	URI string
+}
+
+// NewMobfoxpbBidder Initializes the Bidder
+func NewMobfoxpbBidder(endpoint string) *MobfoxpbAdapter {
+	return &MobfoxpbAdapter{
+		URI: endpoint,
+	}
+}
+
+// MakeRequests create bid request for mobfoxpb demand
+func (a *MobfoxpbAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errs []error
+	var err error
+	var tagID string
+
+	var adapterRequests []*adapters.RequestData
+
+	reqCopy := *request
+	for _, imp := range request.Imp {
+		reqCopy.Imp = []openrtb.Imp{imp}
+
+		tagID, err = jsonparser.GetString(reqCopy.Imp[0].Ext, "bidder", "TagID")
+		if err != nil {
+			errs = append(errs, err)
+			continue
+		}
+
+		reqCopy.Imp[0].TagID = tagID
+
+		adapterReq, errors := a.makeRequest(&reqCopy)
+		if adapterReq != nil {
+			adapterRequests = append(adapterRequests, adapterReq)
+		}
+		errs = append(errs, errors...)
+	}
+	return adapterRequests, errs
+}
+
+func (a *MobfoxpbAdapter) makeRequest(request *openrtb.BidRequest) (*adapters.RequestData, []error) {
+
+	var errs []error
```

Comment:
Does this method really needs to return an error slice? The only place it potentially returns an error is from the JSON Marshal and that's a single error instance. It will be easier to just return `error` rather than a slice of `error`